mods/ra/rules/player.yaml

```diff
@@ -65,4 +65,7 @@ Player:
 		Prerequisites: techlevel.infonly, techlevel.low, techlevel.medium, techlevel.unrestricted
 	GlobalUpgradeManager:
 	EnemyWatcher:
+	ProductionIconOverlay:
+		Offset: -7, 6
```

Comment:
I'd say take @tomson26's chevron from http://www.ppmforums.com/viewtopic.php?p=508926#508926 and place it to the top left corner.